geode-docs/tools_modules/gfsh/command-pages/start.html.md.erb

```diff
@@ -605,8 +605,8 @@ start server --name=value [--assign-buckets(=value)] [--bind-address=value]
 </tr>
 <tr class="odd">
 <td><span class="keyword parmname">\-\-server-bind-address</span></td>
-<td>IP address on which the Server will be bound.</td>
-<td>binds to all addresses</td>
+<td>Overrides the <code>bind-address</code> on which this server will listen for client connections. Set this option in a multi-homed server environment to distinguish communications from clients. Setting a value of the empty string (&quot;&quot;) uses the value of <code>bind-address</code>.</td>
```

Comment:
> Set this option in a multi-homed server environment to distinguish communications from clients.

This sentence is vague about what this property actually does. Maybe something like "Set this option if you want clients to connect to a different interface than what is specified in bind-adress."